resolvers/__tests__/context.spec.js

```diff
@@ -0,0 +1,83 @@
+const { setContext } = require('../context')
+const { fakeCookie } = require('../stubs')
+const { User } = require('../../models/User')
+
+afterEach(() => {
+  jest.clearAllMocks()
+})
+
+beforeEach(async () => {
+  await User.query().delete()
+})
+
+const getAllUsersCount = async () => {
+  const usersCount = await User.query().count('id')
+
+  return Number(usersCount[0].count)
+}
+
+test('returns an object with empty user when no auth cookie found', async () => {
+  const consoleError = jest.fn()
+  jest.spyOn(console, 'error').mockImplementation(consoleError)
```

Comment:
I don't think this is needed, if we change how we are logging errors this test will break.